### ISSUE TITLE
(hostsman) Not the last version.

### ISSUE THREAD

**Issue by Noruas29** [chocolatey-community/chocolatey-coreteampackages#1292]
On official website, last version of Hostsman is 4.7.105 but when you download the "No Installer" version of this soft, you can see that the version number is 4.7.106.

Could you update this package with the last version ?

**Comment by majkinetor** [comment]
We can't make this work. 
Its not available on typical location: http://hostsman2.it-mate.co.uk

Site doesn't work.

Upstream problem. Author needs to fix it and provide downloads via usual means or provide new place.

This is automatic package and requires stable download location.

**Comment by Noruas29** [comment]
But it's not offcial website ?
http://www.abelhadigital.com/hostsman/

**Comment by majkinetor** [comment]
Yes, its official, site works now. I'll add this for up for grabs.

**Comment by Noruas29** [comment]
Thanks a lot and thanks for your work.
